code/js/rcalc-loader.js

```diff
@@ -0,0 +1,15 @@
+// loader injects js files in to the html file <head> tag
+function loadJsScript(jsFile) {
```

Comment:
It is path to js file. Not jsFile.

Reply:
parameter in the function changed from jsFile to jsFilePath